client/kafka/kafka.go

```diff
@@ -98,11 +99,28 @@ func (ap *AsyncProducer) Error() <-chan error {
 	return ap.chErr
 }
 
-// Close gracefully the producer.
+// ActiveBrokers returns a list of active brokers' addresses.
+func (ap *AsyncProducer) ActiveBrokers() []string {
+	brokers := ap.prodClient.Brokers()
+	activeBrokerAddresses := make([]string, len(brokers))
+	for i, b := range brokers {
```

Comment:
@drakos74 to not loose your review feedback from the closed PR: please make me an example how to use `copy` here, I cannot see it :relaxed:

Reply:
thnx @gm42 ! I meant initially to just return the whole Broker object instead of the string, but looking closer, i noticed how big this actuallly is, so pls ignore my comment after all ;)

Reply:
We could do that, but we pass in brokers as strings in patron - hence I did not want to pass them as the fully-qualified broker